aruodas/aruodasbutai.csv

```diff
@@ -0,0 +1,1599 @@
+Pavadinimas,Rajonas,Gatve,Kaina,Visa_info,Dydis,Kambariai,Metai,Aukstas/is,Nuoroda

```

Comment:
this csv should not be part of the git repo. At best you could have it with only the first two lines to show structure, but a git repo is for the code, never for data. 
Move these to a folder `data/` that will be in your gitignore. See for example https://www.pluralsight.com/guides/how-to-use-gitignore-file for its syntax